### PR TITLE
[console] Improvements to find functionality

### LINKED ISSUE #968: [FEATURE][console] Improvements to find functionality
Original commit: https://github.com/qgis/QGIS/commit/99210ecfd713f4380125c4ccfa0908142c8ee3a9 by nyalldawson
- add key shortcut to open find widget, set focus to find text input
  automatically (uses default shortcut for DE, eg usually Ctrl+F)
- add key shortcuts for find next/previous (using default DE
  shortcut, eg Ctrl+G/Shift+Ctrl+G)
- automatically find first match when typing in find box
- set initial find string to selection when opening find
- pressing Escape closes the find bar

### PR DESCRIPTION
**Pull request by yjacolin**
Fix #968

### CODE REVIEW

File: source/docs/user_manual/plugins/python_console.rst [hunk 1]
```diff
@@ -75,7 +76,12 @@ Main features are:
 * Save and clear the command history accessing from context menu of input panel.
   The history will be saved into the file ~/.qgis2/console_history.txt.
```
Comment (yjacolin) [inline source/docs/user_manual/plugins/python_console.rst]: in *nix system, may be add something for our Windows users?

**Comment by yjacolin** [comment]
@DelazJ what you mean by 'already commited'? This is a PR :)

**Comment by DelazJ** [comment]
My bad, indeed it's a PR. It might be because I reached the page through the commit link and that looking to the link it was from qgis repo and not @yjacolin one (and it was late...)

File: source/docs/user_manual/plugins/python_console.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+   .. contents::
+      :local:
+
+As you will see later in this chapter, QGIS has been designed with a plugin
+architecture. Pluginis are written in Python, a very famous language in the
+geospatial world.
+
+QGIS brings a Python API to let the user interact with its objects (layers,
+feature or interface). To help the user, QGIS has a Python console. To open the
+console go to :menuselection:`Plugins --> Python console`` (:kbd:`Ctrl+Alt+P`).
```
Comment (DelazJ) [inline source/docs/user_manual/plugins/python_console.rst]: double ` after console

File: source/docs/user_manual/plugins/python_console.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
+.. only:: html
+
+   |updatedisclaimer|
+
+.. index::
+   single: Pyhon
```
Comment (DelazJ) [inline source/docs/user_manual/plugins/python_console.rst]: Python

**Pull request merged**